man/systemctl.xml

```diff
@@ -2067,21 +2067,9 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           is defined. If omitted, defaults to
           <option>all</option>.</para>
         </listitem>
-
       </varlistentry>
 
-      <varlistentry>
-        <term><option>-s</option></term>
-        <term><option>--signal=</option></term>
-
-        <listitem>
-          <para>When used with <command>kill</command>, choose which
-          signal to send to selected processes. Must be one of the
-          well-known signal specifiers such as <constant>SIGTERM</constant>, <constant>SIGINT</constant> or
-          <constant>SIGSTOP</constant>. If omitted, defaults to
-          <option>SIGTERM</option>.</para>
-        </listitem>
-      </varlistentry>
+      <xi:include href="standard-options.xml" xpointer="signal" />
```

Comment:
this isn't added to loginctl.xml in this patch so far?

Reply:
loginctl needs a slightly different text. I don't think docbooks allows us to do it any reasonable way, so the text is duplicated.